src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(int side, String color) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double getArea() {
+        return side * side;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle, area: " + getArea() + " sq. units, side: "
+                + side + " units, color: " + getColor());

```

Comment:
The message in the `draw` method incorrectly states the figure as 'rectangle', whereas it should be 'square'. Also, consider using `name()` for getting String representation of enum constants instead of `getColor()` if `color` is of an enum type.